demisto_sdk/tests/integration_tests/find_dependencies_integration_test.py

```diff
@@ -56,6 +57,7 @@ def test_integration_find_dependencies_sanity(self, mocker, repo):
         mocker.patch.dict(os.environ, {'DEMISTO_SDK_ID_SET_REFRESH_INTERVAL': '-1'})
         pack = repo.create_pack('FindDependencyPack')
         integration = pack.create_integration('integration')
+        integration.create_default_integration()
```

Comment:
why did you modify this test?

Reply:
The current test is not passing on master branch (not sure how this is possible), but adding this line, which populates the created integration in the previous line, assures the test passes. This was added based on this [PR](https://github.com/demisto/demisto-sdk/pull/1606/files) (which is pending merge).